.env.example

```diff
@@ -36,10 +36,10 @@ REACT_APP_REALITY_CONTRACT_ADDRESS_FOR_RINKEBY='0x3D00D77ee771405628a4bA4913175E
 REACT_APP_REALITY_CONTRACT_ADDRESS_FOR_GANACHE='0xcfeb869f69431e42cdb54a4f4f105c19c080a601'
 REACT_APP_REALITY_CONTRACT_ADDRESS_FOR_XDAI='0x79e32aE03fb27B07C89c0c568F80287C01ca2E57'
 
-REACT_APP_WRAPPED_1155_FACTORY_CONTRACT_ADDRESS_FOR_MAINNET='0xDE6943f3717738038159a406FF157d4eb3238c1B'
-REACT_APP_WRAPPED_1155_FACTORY_CONTRACT_ADDRESS_FOR_RINKEBY='0xDE6943f3717738038159a406FF157d4eb3238c1B'
-REACT_APP_WRAPPED_1155_FACTORY_CONTRACT_ADDRESS_FOR_GANACHE='0xDE6943f3717738038159a406FF157d4eb3238c1B'
-REACT_APP_WRAPPED_1155_FACTORY_CONTRACT_ADDRESS_FOR_XDAI='0xDE6943f3717738038159a406FF157d4eb3238c1B'
+REACT_APP_WRAPPED_1155_FACTORY_CONTRACT_ADDRESS_FOR_MAINNET='0x6846a6e06B88c1449Db0B3cc555d4638eDC1bF37'
```

Comment:
It's missing  https://etherscan.io/address/0x6846a6e06B88c1449Db0B3cc555d4638eDC1bF37

Reply:
wrapped 1155 it's not deployed yet. Once we deploy the contract on mainnet will generate the same address based on the bycode and the singleton contract.

Reply:
The env.example can be updated one the contract PR is merged and deployed.

Reply:
The new wrapper address is `0xc14f5d2B9d6945EF1BA93f8dB20294b90FA5b5b1`